File: Проект Эйлера/python/task_2.py
import time
arr = []
f = [1,2]
arr.append(f[1])
while True:
      f[0] = f[0] + f[1]
      if f[0] % 2 == 0:
            arr.append(f[0])
      f[1] = f[0] + f[1]
      if f[1] % 2 == 0:
            arr.append(f[1])
      time.sleep(0.5)
      print arr
      if f[0] >= 4000000:
            print("gg")
            break
      if f[1] >= 3500000:
            print("gg")
            break
otvet = sum(arr)
print otvet

File: Проект Эйлера/php/task_2.php
<?php

	$arr = [];
	$f = [1,2];
	array_push($arr,$f[1]);
	while(true){
		$f[0] = $f[0] + $f[1];
		if($f[0] % 2 == 0){
			array_push($arr,$f[0]);
		}
		$f[1] = $f[0] + $f[1];
		if($f[1] % 2 == 0){
			array_push($arr,$f[1]);
		}
		print_r ($arr);
		if($f[0] >= 4000000){
			break;
		}
		elseif($f[1] >= 4000000){
			break;
		}
	}
	$sum = array_sum($arr);
	echo $sum;
?>